portfolio/src/main/webapp/script.js

```diff
@@ -73,14 +73,21 @@ function contact() {
 
 /** Creates a map and adds it to the page. */
 function loadMap() {
-  const stanfordGatesBldgLatLng = {lat:37.430175, lng:-122.173388};
+  var desiredLocationLatLng = {lat: 46.233950, lng: 6.055801};
+  if (document.getElementById("locations").value == "Ramen") {
+      desiredLocationLatLng = {lat: 37.445640, lng: -122.160736};
```

Comment:
Here the first level indent is 2 spaces but the second level indent is 4 spaces.

Reply:
fixed! thanks!